src/app/shared/version-picker/version-picker.html

```diff
@@ -3,7 +3,7 @@
   {{materialVersion}}
   <mat-icon>arrow_drop_down</mat-icon>
 </button>
-<mat-menu #versionPicker="matMenu">
+<mat-menu #versionPicker="matMenu" xPosition="before">
```

Comment:
Not necessary for the reduced motion fix, but I think that it looks better when the version picker opens in the same direction as the theme picker.